modules/framework/galasa-parent/dev.galasa.framework.k8s.controller/src/main/java/dev/galasa/framework/k8s/controller/TestPodScheduler.java

```diff
@@ -501,21 +491,6 @@ public static void filterTerminated(@NotNull List<V1Pod> pods) {
         }
     }
 
-    private static class Pool implements Comparable<Pool> {
```

Comment:
This method was not being used. Removing for clarity.

Reply:
OK